### PR TITLE
Train 813 service catalog course creation

### PR DESCRIPTION
**Pull request by spottsdd**
Initial creation of assets for the Service Catalog course

### CODE REVIEW

File: courses/service-catalog/incidents/inc-frontend3.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (stevedatadog) [inline courses/service-catalog/incidents/inc-frontend3.json:1]: This also looks the same?

File: courses/service-catalog/incidents/inc-frontend4.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (stevedatadog) [inline courses/service-catalog/incidents/inc-frontend4.json:1]: Same? I feel like I'm missing something.

File: courses/service-catalog/incidents/inc-tester2.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (stevedatadog) [inline courses/service-catalog/incidents/inc-tester2.json:1]: Again, same file?

File: courses/service-catalog/services/backend/Gemfile [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+gem 'activerecord-nulldb-adapter'
+
+# improved JSON rendering performance
+gem 'oj'
+
+# Fix SCSS errors with Ruby 3 on MacOS
+gem 'sassc', github: 'sass/sassc-ruby', group: :development
+
+gem 'ddtrace', require: 'ddtrace/auto_instrument'
```
Comment (stevedatadog) [inline courses/service-catalog/services/backend/Gemfile:106]: Is this the only difference from the default Gemfile? If so, is this a job for `sed`?

File: courses/service-catalog/services/worker/Gemfile [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+gem 'activerecord-nulldb-adapter'
+
+# improved JSON rendering performance
+gem 'oj'
+
+# Fix SCSS errors with Ruby 3 on MacOS
+gem 'sassc', github: 'sass/sassc-ruby', group: :development
+
+gem 'ddtrace', require: 'ddtrace/auto_instrument'
```
Comment (stevedatadog) [inline courses/service-catalog/services/worker/Gemfile:106]: `sed`?

File: courses/service-catalog/status.conf [hunk 6]
```diff
@@ -0,0 +1,24 @@
+server {
```
Comment (stevedatadog) [inline courses/service-catalog/status.conf:1]: `nginx_status.conf`?

**Review by stevedatadog — APPROVED** [review]

File: courses/service-catalog/create-incidents.sh [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+ALL_MONITORS=$(
+    ${CMD_CURL} -X GET "${API_URL}" \
+    -H "Content-Type: application/json" \
+    -H "DD-API-KEY: ${DD_API_KEY}" \
+    -H "DD-APPLICATION-KEY: ${DD_APP_KEY}"
+)
+
+# Check if the monitor already exists. This avoids duplicates when the lab is ran multiple times.
+# If it doesn't exist, create it.
+for key in ${!MONITOR_ARRAY[@]}
```
[the remark this answers is not in the document]
Comment (spottsdd) [reply courses/service-catalog/create-incidents.sh]: replaced loop in all scripts with this recommendation

File: courses/service-catalog/create-synthetics.sh [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+for key in "${!GLOBAL_VAR_ARRAY[@]}"
+do
+  GLOBAL_ID_ARRAY[$key]=$(
+    ${CMD_JQ} --null-input --raw-output --argjson data "${ALL_GLOBAL_VARS}" \
+    '$data.variables[] | select(.name=="'$key'").id'
+  )
+done
+
+# function to update variable value
+update_global_var() {
```
[the remark this answers is not in the document]
Comment (spottsdd) [reply courses/service-catalog/create-synthetics.sh]: created new function to combine behavior.

**Pull request merged**